soccer_vision_attribute_msgs/README.md

```diff
@@ -0,0 +1,12 @@
+# soccer_vision_attribute_msgs
+
+This package provides messages for attributes of visually detected objects on a soccer field, such as balls, field lines and robots
```

Comment:
```suggestion
This package provides messages for attributes of visually detected objects on a soccer field, such as balls, field lines and robots.
```
I don't know how to phrase it better, but at first I had a hard understanding, what this whole package was far. Now I know better, but it was not intuitive.

Reply:
I agree, I've updated to include a better description of the package.